update_tag_sample_page.py

```diff
@@ -0,0 +1,50 @@
+import sys
+import argparse
+from rest import McM
+sys.path.append('/afs/cern.ch/cms/PPD/PdmV/tools/McM/')
+
+parser = argparse.ArgumentParser(description='Tag samples for an analysis subgroup')
+parser.add_argument('--input_file', default='text_not_existing.txt', required=True,
+                    help='input file where miniAOD names are stored')
+parser.add_argument('--tag', default='default_tag', required=True,
+                    help='tag to include in McM for the specified samples')
+parser.add_argument('--dev', default=True, required=False,
+                    help='Run in dev. McM')
+
+args = parser.parse_args()
+
+sys.stdout.flush()
+
+mcm = McM(dev=False)
+
+if vars(args)['dev']:
+    mcm = McM(dev=True, cookie='dev_cookie.txt')
+
+#read the input file and collect the inputs
+file_inputs = open(vars(args)['input_file'], 'r')
+
+samples = file_inputs.readlines()
+
+for sample in samples:
+
+    #get requests from McM
+    query_tag = 'produce=%s' % (sample.rstrip())
```

Comment:
You can use `.strip()` to remove whitespaces from both beginning and end of the `sample` variable.

Reply:
I want to keep the rstrip because it should also remove new lines for the last sample in the text file